SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -176,13 +181,13 @@ final class SwedbankPaySDKViewModel: NSObject {
             
             let json = try! JSONEncoder().encode(["paymentorder": paymentOrder])
             
-            var request = try! URLRequest(url: endPointUrl, method: .post, headers: self?.configuration?.headers)
+            var request = try! URLRequest(url: endPointUrl, method: .post, headers: self?.configuration?.afHeaders)
```

Comment:
Codacy found an issue: [Force tries should be avoided.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)